wurst/systems/core/ItemAmount.wurst

```diff
@@ -0,0 +1,29 @@
+package ItemAmount
+
+import ClosureEvents
+import StackNSplit
+
+function onDrop(item itm)
+    if isItemStackable(itm.getTypeId())
+        var name = itm.getName()
+        int foundIndex = name.lastIndexOf(" (x")
```

Comment:
Abstract lines 9 - 11 into a function. You're doing the exact same thing in the `onPickup`.

Reply:
updated